Examples/Python/tests/conftest.py

```diff
@@ -25,14 +25,16 @@
 import acts.examples
 
 try:
-    acts.logging.setFailureThreshold(acts.logging.WARNING)
+    if acts.logging.getFailureThreshold() != acts.logging.WARNING:
+        acts.logging.setFailureThreshold(acts.logging.WARNING)
 except RuntimeError:
     # Repackage with different error string
-    raise RuntimeError(
+    warnings.warn(
         "Runtime log failure threshold could not be set. "
         "Compile-time value is probably set via CMake, i.e. "
-        f"`ACTS_LOG_FAILURE_THRESHOLD={acts.logging.getFailureThreshold().name}` is set. The "
-        "pytest test-suite will not work in this configuration."
+        f"`ACTS_LOG_FAILURE_THRESHOLD={acts.logging.getFailureThreshold().name}` is set, "
+        "or `ACTS_ENABLE_LOG_FAILURE_THRESHOLD=OFF`. "
+        "The pytest test-suite can produce false results in this configuration"
```

Comment:
I guess many users who want to continue will want to know if it will produce false-positive info, or false-negative warnings. I guess it's usually the latter (with `ACTS_ENABLE_LOG_FAILURE_THRESHOLD=OFF`).
Can we adjust the message to say which it is for newcomers? I guess you can do that by comparing `acts.logging.getFailureThreshold() < acts.logging.WARNING` and adjusting the text appropriately.

Reply:
Sure, if you think that helps.

Reply:
How about this, @timadye ?

Reply:
Looks good. Thanks!